iree_tests/benchmarks/sdxl/conftest.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+
+def pytest_addoption(parser):
+    parser.addoption("--goldentime-rocm-e2e-ms", action="store", default=1661.5, type=float, help="Golden time to test benchmark")
+    parser.addoption("--goldentime-rocm-unet-ms", action="store", default=450.5, type=float, help="Golden time to test benchmark")
+    parser.addoption("--goldentime-rocm-clip-ms", action="store", default=19, type=float, help="Golden time to test benchmark")
+    parser.addoption("--goldentime-rocm-vae-ms", action="store", default=288.5, type=float, help="Golden time to test benchmark")
+    parser.addoption("--goldendispatch-rocm-unet", action="store", default=1718, type=int, help="Golden dispatch count to test benchmark")
+    parser.addoption("--goldendispatch-rocm-clip", action="store", default=1571, type=int, help="Golden dispatch count to test benchmark")
+    parser.addoption("--goldendispatch-rocm-vae", action="store", default=250, type=int, help="Golden dispatch count to test benchmark")
+    parser.addoption("--goldensize-rocm-unet-bytes", action="store", default=2088217, type=int, help="Golden vmfb size to test benchmark")
+    parser.addoption("--goldensize-rocm-clip-bytes", action="store", default=785493, type=int, help="Golden vmfb size to test benchmark")
+    parser.addoption("--goldensize-rocm-vae-bytes", action="store", default=762067, type=int, help="Golden vmfb size to test benchmark")
+    parser.addoption("--gpu-number", action="store", default=6, type=int, help="IREE GPU device number to test on")
```

Comment:
I'd default this to 0 instead of 6.

Reply:
Others also use this node. I think 6 is best bet to not have any interference

Reply:
Yeah - using node 6 on the current test machine is all fine, but I don't think that should be baked into the script that gets used across repositories as a default.